src/components/Person/Person.jsx

```diff
@@ -1 +1,46 @@
-// Write code here
+// filename: Person.jsx
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export function Person({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) {
+
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">
+          {`My name is ${name}`}
+        </h2>
+        <p className="Person__age">
+          {age && (`I am ${age}`)}
+        </p>
```

Comment:
so, here you still generate the age element, but it has zero-value
![Olya is zero-aged](https://user-images.githubusercontent.com/39308717/172112057-adc0c48b-3cd3-4bb6-b2c1-a199c1abb0e4.png)

you should not generate the whole element if there's no age
```suggestion
        {age && (
          <p className="Person__age">
            `I am ${age}`
          </p>)
        }
```